locales/ara.json

```diff
@@ -337,58 +399,58 @@
     "sharingVc": "مشاركة بطاقة ",
     "errors": {
       "locationDisabled": {
-        "message": "يجب تمكين خدمات الموقع لمشاركة البطاقة",
+        "message": "يجب تمكين خدمات الموقع لوظيفة المسح",
         "button": "تمكين خدمات الموقع"
       },
       "locationDenied": {
-        "message": "مطلوب إذن الموقع لمشاركة البطاقة",
+        "message": "إذن الموقع مطلوب لوظيفة المسح",
         "button": "الموقع"
-      },
-      "nearbyDevicesPermissionDenied": {
-        "message": "مطلوب إذن \"الأجهزة المجاورة\" لتتمكن من مشاركة البطاقة",
-        "button": "السماح إذن"
       }
     },
     "status": {
-      "connecting": "ارٍ التوصيل...",
+      "inProgress": "في تَقَدم",
+      "establishingConnection": "اجراء اتصال",
+      "sharingInProgress": "جاري المشاركة",
       "connectingTimeout": "يستغرق إنشاء الاتصال بعض الوقت. هل الجهاز الآخر مفتوح للاتصالات؟",
       "exchangingDeviceInfo": "تبادل معلومات الجهاز ...",
       "exchangingDeviceInfoTimeout": "يستغرق الأمر بعض الوقت لتبادل معلومات الجهاز. قد تضطر إلى إعادة الاتصال.",
-      "invalid": "غير صالح",
+      "invalid": "رمز الاستجابة السريعة غير صالح",
       "offline": "يرجى الاتصال بالإنترنت لمسح رموز QR ضوئيًا باستخدام وضع المشاركة عبر الإنترنت",
-      "sent": "تم إرسال البطاقة ...",
-      "sentHint": "في انتظار أن يحفظ المستلم بطاقة  أو يتجاهلها",
+      "sent": "تم إرسال بطاقة ...",
+      "sentHint": "في انتظار أن يحفظ المستلم بطاقة أو يتجاهلها",
       "sharing": {
         "title": "مشاركة...",
         "hint": "الرجاء الانتظار حتى يقبل الجهاز المتلقي المشاركة أو يرفضها.",
         "timeoutHint": "تستغرق مشاركتها وقتًا أطول من المتوقع. قد تكون هناك مشكلة في الاتصال."
       },
       "accepted": {
-        "title": "نجاح!",
-        "message": "تمت مشاركة بطاقة  بنجاح مع المدقق"
+        "title": "تم مشاركة المعرف بنجاح",
+        "message": "تمت مشاركة بطاقة بنجاح مع المدقق",
+        "gotohome": "اذهب الى المنزل"
       },
       "rejected": {
         "title": "يلاحظ",
-        "message": "تم تجاهل بطاقة  بواسطة المدقق"
+        "message": "تم تجاهل بطاقة بواسطة المدقق"
       },
       "bleError": {
-        "title": "فشل التحويل",
-        "message": "حدث خطأ أثناء نقل بطاقة . حاول مرة اخرى.",
-        "hint": "خطأ: {{code}}"
+        "title": "فشل التحويل!",
+        "message": "حدث خطأ أثناء نقل بطاقة. حاول مرة اخرى."
       }
     }
   },
   "SelectVcOverlay": {
-    "header": "يشارك بطاقة ",
-    "chooseVc": "اختر بطاقة  الذي ترغب في المشاركة معه",
+    "header": "يشارك بطاقة",
+    "chooseVc": "اختر بطاقة الذي ترغب في المشاركة معه",
     "share": "شارك",
     "verifyAndShare": "التحقق من الهوية والمشاركة"
   },
   "SendVcScreen": {
     "reasonForSharing": "سبب المشاركة (اختياري)",
     "acceptRequest": "يشارك",
-    "acceptRequestAndVerify": "شارك مع سيلفي",
+    "acceptRequestAndVerify": "شارك مع Selfie",
```

Comment:
can you please translate this as well, please check in the current changes in the upstream

Reply:
Changed the above one. Some of the words were not getting in Google Translate too.